tests/e2e/tests/critical-path/memory-efficiency/top-keys-table.e2e.ts

```diff
@@ -0,0 +1,67 @@
+import { Chance } from 'chance';
+import { Selector } from 'testcafe';
+import { MyRedisDatabasePage, MemoryEfficiencyPage, BrowserPage, CliPage } from '../../../pageObjects';
+import { rte } from '../../../helpers/constants';
+import { acceptLicenseTermsAndAddDatabaseApi } from '../../../helpers/database';
+import { commonUrl, ossStandaloneConfig } from '../../../helpers/conf';
+import { deleteStandaloneDatabaseApi } from '../../../helpers/api/api-database';
+import { populateDBWithHashes } from '../../../helpers/keys';
+
+const memoryEfficiencyPage = new MemoryEfficiencyPage();
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const browserPage = new BrowserPage();
+const cliPage = new CliPage();
+const chance = new Chance();
+
+const keyToAddParameters = { keysCount: 13, keyNameStartWith: 'hashKey'};
+const members = [...Array(100).keys()].toString().replace(/,/g, ' '); // The smallest key
+const keyNamesMemory = ['string', 'list', 'bloom', 'set'];
+const keyNamesLength = ['string', 'set', 'list'];
+const mbloomCommand = 'bf.add bloom 1';
+const listCommand = `rpush list ${members.slice(0, -3)}`;
+const stringCommand = `set string "${chance.paragraph({ sentences: 100 })}"`; // The biggest key
+const setCommand = `sadd set ${members}`; // Middle key
+
+fixture `Memory Efficiency Top Keys Table`
+    .meta({ type: 'critical_path', rte: rte.standalone })
+    .page(commonUrl);
+test
+    .before(async t => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+        // Create keys
+        await populateDBWithHashes(ossStandaloneConfig.host, ossStandaloneConfig.port, keyToAddParameters);
+        // Go to Analysis Tools page
+        await t.click(myRedisDatabasePage.analysisPageButton);
+    })
+    .after(async() => {
+        await cliPage.sendCommandInCli('flushdb');
```

Comment:
I think that it's not good to delete all keys in this db if in future we will make testcafe parallelizm
also we have this function to delete all keys from db:
`await deleteAllKeysFromDB(dbParameters.host, dbParameters.port);`

so it's better to use ossStandaloneRedisearch db here for me

Reply:
We have 4 different threads and for each thread we create new RTE and tests are not interacted between each other so I think that flashing all db in one command will not increase tests run. Vise verse I would delete via UI ~20 keys.
Function "deleteAllKeysFromDB" is also based on flushdb command

Reply:
fixed